examples/models/rex_quadrotor_model.jl

```diff
@@ -0,0 +1,176 @@
+using Rotations
+using ForwardDiff
+using LinearAlgebra
+# using ControlSystems
+# using JSON
+using StaticArrays
+using Printf
+using RobotDynamics
+
+const RD = RobotDynamics
+
+const quad_mass = 2.0
+const quad_inertia = SMatrix{3,3,Float64,9}([0.01566089 0.00000318037 0; 0.00000318037 0.01562078 0; 0 0 0.02226868])
+const quad_motor_kf = 0.0244101
+const quad_motor_bf = -30.48576
+const quad_motor_km = 0.00029958
+const quad_motor_bm = -0.367697
+const quad_arm_len = 0.28
+const quad_min_throttle = 1148.0
+const quad_max_throttle = 1832.0
+
+RD.@autodiff struct RexQuadrotor{R} <: RD.RigidBody{R}
```

Comment:
sorry, one more tweak here. since you've hard-coded the MRP in the functions below you should just restrict it here, e.g.:
```
struct RexQuadrotor <: RD.RigidBody{MRP}
...
end
```